parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -248,8 +272,7 @@ installKubeletKubectlAndKubeProxy() {
             if [[ "$CONTAINER_RUNTIME" == "docker" ]]; then
                 extractHyperkube "docker"
             else
-                installImg
-                extractHyperkube "img"
+                extractHyperkube "containerd"
```

Comment:
seems weird we pass this to extractHyperkube which just converts it back to `ctr`

Reply:
yea extractHyperkube has a simple mutate logic from contaienrd => ctr, mostly to catch script errors like this one. changing to `extractHyperkube ctr`